query-engine/connectors/sql-query-connector/src/cursor_condition.rs

```diff
@@ -1,46 +1,80 @@
-use crate::query_arguments_ext::QueryArgumentsExt;
+use crate::{ordering::OrderingJoins, query_arguments_ext::QueryArgumentsExt};
 use connector_interface::QueryArguments;
 use prisma_models::*;
 use quaint::ast::*;
 
 static ORDER_TABLE_ALIAS: &str = "order_cmp";
 
+type AliasedScalar = (ScalarFieldRef, String);
+type MaybeAliasedScalar = (ScalarFieldRef, Option<String>);
+
+#[derive(Debug)]
+struct CursorOrderDefinition {
+    // Field on which to perform the ordering
+    pub(crate) field_aliased: AliasedScalar,
+    // Direction of the sort
+    pub(crate) sort_order: SortOrder,
+    // Final column identifier to be used for the scalar field to order by
+    pub(crate) order_column: Column<'static>,
+    // Foreign keys of the relations on which the order is performed
+    pub(crate) fks: Option<Vec<MaybeAliasedScalar>>,
+}
```

Comment:
Renamed it from `OrderDefinition` to `CursorOrderDefinition` 😂🤷‍♂️. Hope you enjoy my creative naming skill 😅

Reply:
Better than before :)